src/Algebra/Graph.hs

```diff
@@ -303,7 +306,7 @@ vertices = overlays . map vertex
 -- 'edgeCount' . edges == 'length' . 'Data.List.nub'
 -- @
 edges :: [(a, a)] -> Graph a
-edges = overlays . map (uncurry edge)
+edges = overlays . map edge'
```

Comment:
This change seems to be unrelated to the rest of the PR? Do you really need this? Presumable, fusion doesn't care if the function you map is `edge'` or `uncurry edge`.

Reply:
Indeed, I was too happy to find something here ^^. I will open a separate issue